sentinel-adapter/pom.xml

```diff
@@ -18,12 +18,12 @@
         <module>sentinel-web-servlet</module>
         <module>sentinel-dubbo-adapter</module>
         <module>sentinel-apache-dubbo-adapter</module>
-        <module>sentinel-grpc-adapter</module>
```

Comment:
What is the reason for deleting this `sentinel-grpc-adapter` module?

Reply:
Miss class in module: sentinel-grpc-adapter

Reply:
Has been modified